rubix-tests/src/test/robotframework/acceptance/CacheEviction.robot

```diff
@@ -0,0 +1,68 @@
+*** Settings ***
+Documentation   Rubix Integration Test PoC
+Resource        bks.robot
+
+*** Variables ***
+${METRIC_EVICTION}  rubix.bookkeeper.count.cache_eviction
+
+${REMOTE_PATH}      ${CURDIR}${/}rubixIntegrationTestFile
+${FILE_LENGTH}      1048576
+${LAST_MODIFIED}    1514764800
+${START_BLOCK}      0
+${END_BLOCK}        1048576
+${CLUSTER_TYPE}     3   # TEST_CLUSTER_MANAGER
+
+${CACHE_DIR_PFX}    /Users/jordanw/RobotTest/dtest/
```

Comment:
This needs to be some system level directory. The same way we create directories in our unit tests

Reply:
Updated to use OS's temporary directory for caching & holding data files